### ISSUE TITLE
Selected item replaces from last space rather than the last trigger

### ISSUE THREAD

**Issue by ellerynz** [webscopeio/react-textarea-autocomplete#50]
Hi there,

Thanks for putting this together! I was wondering if there is a way to prevent the following behaviour:

![screencast 2018-02-19 15-41](https://user-images.githubusercontent.com/1263717/36362218-7027662a-1597-11e8-8469-e6a8306cc81a.gif)

```
Hello :world # Ouputs => Hello :earth_americas:
Hello:world # Outputs => :earth_americas: but expected Hello:earth_americas:
```

Looks like everything from/including the last space is replaced but I was expect everything from/including the last _trigger_ to be replaced.

This seems related to the resolved issue #43.

**Comment by jukben** [comment]
👋Hi! This is also reproducible on https://codepen.io/jukben/pen/bYZqvR which is always the last version. It's definitely a bug, sorry about it I will take a look ASAP. 💪 Thanks for reporting!

**Comment by jukben** [comment]
I have good news @ellerynz. This should be fixed in v1.5.2! Take it while it's hot. 🔥

Please, if you like the package, like it, share it. Your support means a lot to me. Thanks again for creating an issue. 💪

**Comment by ellerynz** [comment]
@jukben Nice speedy fix! Thanks!

![cheers](https://user-images.githubusercontent.com/1263717/36399199-a290b2fe-162f-11e8-9320-e14d7e7f652a.gif)
